discovery_engine/lib/src/domain/document_manager.dart

```diff
@@ -81,6 +81,7 @@ class DocumentManager {
     DocumentId id,
     DocumentViewMode mode,
     int sec,
+    DocumentFeedback feedback,
```

Comment:
I don't think we should add `DocumentFeedback` to the list of args here or to the `DocumentTimeSpent` event. Insteas we should use the `DocumentId` to get a `Document` from the database, and get the feedback from that document.

```dart
final doc = await _documentRepo.fetchById(id);
/*...*/
_engine.timeLogged(
  id,
  smbertEmbedding: activeData.smbertEmbedding,
  seconds: sumDuration,
  reaction: doc.feedback, // access the feedback from the document
);
```

Reply:
That makes sense 👍 I will implement it like this
Is it possible that a document does not exist?

Reply:
Done also I added two tests